index.d.ts

```diff
@@ -61,7 +61,7 @@ declare module 'connected-react-router' {
   export function createMatchSelector<
     S extends RouterRootState, Params extends { [K in keyof Params]?: string }
   >(path: string): matchSelectorFn<S, Params>;
-  export function onLocationChanged(location: Location, action: RouterActionType, isFirstRendering: boolean = false)
+  export function onLocationChanged(location: Location, action: RouterActionType, isFirstRendering: boolean)
```

Comment:
this should be marked as optional parameter `isFirstRendering?: boolean`

Reply:
Yes. It needs to be optional.